lib/scylla_cloud.py

```diff
@@ -500,6 +512,14 @@ def io_setup():
         return run('/opt/scylladb/scylla-machine-image/scylla_cloud_io_setup', shell=True, check=True)
 
 class aws_instance:
+    @property
+    def user_data(self):
+        encoded_user_data = self.__instance_metadata("/compute/userData")
+        if not encoded_user_data:
+            return ''
+        return base64.b64decode(encoded_user_data).decode()
```

Comment:
This looks like it should be in azure class, shouldn't it?

Reply:
@syuu1228 , was this fixed? i don't see any changes with this change

Reply:
@fgelcer it fixed, see the change https://github.com/scylladb/scylla-machine-image/compare/7ee809bb2423717b025d226f2d8a413836fa7f84..7df851bd64107b2967f3fb54842bd049fe555ac2

Reply:
great, thank you...
in that case, LGTM